Source/WebKit/UIProcess/mac/WKImmediateActionController.mm

```diff
@@ -175,6 +175,12 @@ - (void)immediateActionRecognizerWillPrepare:(NSImmediateActionGestureRecognizer
     _state = WebKit::ImmediateActionState::Pending;
     immediateActionRecognizer.animationController = nil;
 
+    if (!_page->mainFrame())
```

Comment:
Should we bail earlier? Also, if we do, I think we need to keep a smart pointer of whatever kind on the stack, per the new rules.

(As it stands it's ok since you're only using it for a trivial getter but if you move the check up higher the object could disappear between then and the use below.)

Reply:
I thought about bailing earlier, but I wasn't sure what the standard was.

Reply:
If we don't _have_ to check earlier, it makes sense to me to just do it here, but I will change it if required.